sympy/physics/continuum_mechanics/beam.py

```diff
@@ -396,6 +396,12 @@ def apply_load(self, value, start, order, end=None):
         start = sympify(start)
         order = sympify(order)
 
+        if start.is_real:
```

Comment:
@smichr I think doing this way will be better because if start is_real then definitely end will also be real.